includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -128,11 +145,24 @@ public function set_story_ad_slot_id( $ad_slot_id ) {
 	private function render_story_auto_ads() {
 		$config = array(
 			'ad-attributes' => array(
-				'type'           => 'adsense',
-				'data-ad-client' => $this->tag_id,
-				'data-ad-slot'   => $this->story_ad_slot_id,
+				'type' => 'adsense',
 			),
 		);
+
+		$amp_auto_ads_opt = array(
+			'data-ad-client' => $this->tag_id,
+			'data-ad-slot'   => $this->story_ad_slot_id,
+		);
```

Comment:
Same with this array. Let's remove `data-` prefix from array keys and added to filtered array keys to make sure that it adds only `data-*` attributes.

Reply:
Updated @eugene-manuilov Can you check again?
Thanks!